editor/src/components/canvas/ui-jsx-canvas.tsx

```diff
@@ -264,6 +264,7 @@ export const UiJsxCanvas = betterReactMemo(
     }
 
     let metadataContext: UiJsxCanvasContextData = React.useContext(UiJsxCanvasContext)
+    metadataContext.current.spyValues.metadata = {}
```

Comment:
@enidemi I'm very confused! 
I thought we are going to clear the metadata _per scene_, but this seems to clear the entire spy values. isn't this causing problems with scenes that are memoized? if we clear everything here, then we don't even need the cullSpyCollector… am I missing something? or is this code leftover from debugging?

Reply:
I haven't checked the memoization yet, it's true this might cause problems there 😬 
I tried to clear the metadata even if scenes are deleted from the model. With the previous solution those would be kept in the model.

Reply:
but you made this here to get rid of scenes which are deleted! something doesn't add up – I suspect we should also have some scene memoization tests too

https://github.com/concrete-utopia/utopia/pull/1402/files#diff-3ee060e2443faa6e8082bec9c9df5819a3a6d070c17446342ef65e737c9f02ebR2427